Directory.Build.props

```diff
@@ -16,8 +16,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VersionPrefix>8.1.2</VersionPrefix>
-    <VersionSuffix></VersionSuffix>
+    <VersionPrefix>8.1.3</VersionPrefix>
+    <VersionSuffix>preview-1</VersionSuffix>
```

Comment:
Let's use zeros in front of numbers to prevent the issue with the order of preview versions. So this preview version will be `preview-01`.

Reply:
Fixed!!